.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "spec"]
+	path = spec
```

Comment:
Since we're only using purl-spec test data, we should keep it inside `tests/data/`.
```suggestion
	path = tests/data/purl-spec/
```

Reply:
I guess we should not keep the complete spec in tests